src/Tabs/index.jsx

```diff
@@ -43,25 +43,21 @@ class Tabs extends React.Component {
       const labelId = this.genLabelId(i);
 
       return (
-        <li
-          className={styles['nav-item']}
+        <button
```

Comment:
Thoughts on using the `Button` component here?

Reply:
I don't have a preference, but I'm willing to convert it if that makes sense.

Reply:
Personally, my preference would be to use the `Button` component.